LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -56,6 +54,16 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
 
     public func updateUIView(_ textField: UITextField, context: Context) {
         textField.text = text
+        textField.placeholder = placeholder
+        textField.font = font
+        textField.textColor = textColor
+        textField.textAlignment = textAlignment
+        textField.keyboardType = keyboardType
+
+        if didBecomeFirstResponder && !context.coordinator.didBecomeFirstResponder {
```

Comment:
[nit] Can you keep track of whether it did become first responder here, or is the coordinator required?

Reply:
The view itself isn't mutable from this context, so the coordinator (which is a class) is used to track the state change.